smack-extensions/src/test/java/org/jivesoftware/smackx/bytestreams/socks5/Socks5ProxyTest.java

```diff
@@ -102,10 +102,15 @@ public void shouldUseFreePortOnNegativeValues() throws Exception {
     @Test
     public void shouldPreserveAddressOrderOnInsertions() {
         Socks5Proxy proxy = Socks5Proxy.getSocks5Proxy();
-        List<String> addresses = new ArrayList<String>(proxy.getLocalAddresses());
-        addresses.add("1");
-        addresses.add("2");
-        addresses.add("3");
+        List<String> addresses = new ArrayList<>(proxy.getLocalAddresses());
```

Comment:
This should use `LinkedHashSet`, which would reduce the LOC since the `!contains()` check can be avoided.